src/app/public/modules/date-range-picker/date-range-picker.component.scss

```diff
@@ -0,0 +1,40 @@
+@import "~@skyux/theme/scss/mixins";
+@import "~@skyux/theme/scss/_compat/mixins";
+
+:host() {
+  display: flex;
+  padding: 10px;
```

Comment:
I'm seeing a few 10px / 5px properties in here. Should we be using the `sky-padding` variables?

Reply:
We can, I just avoided it since as far as I can recall their fate is still up in the air

Reply:
Oh? I didn't know that. I suppose there's more to the story then... I wouldn't worry about this for now.